src/Facade/Services/TariffService.cs

```diff
@@ -7,14 +7,17 @@
     using Linn.Products.Domain.Linnapps.Products;
     using Linn.Products.Domain.Linnapps.Repositories;
     using Linn.Products.Resources;
+    using Persistence.Linnapps;
 
     public class TariffService : ITariffService
     {
         private readonly ITariffRepository tariffRepository;
+        private readonly ITransactionManager transactionManager;
 
-        public TariffService(ITariffRepository tariffRepository)
+        public TariffService(ITariffRepository tariffRepository, ITransactionManager transactionManager)
```

Comment:
Can this be IRepository<>?

Reply:
It could but it has a custom search so I though this was better